### PR TITLE
feat(backend): extended login failure response

### PR DESCRIPTION
**Pull request by allynt**
Added `is_verified`, `is_approved`, and `accepted_terms` to the `UserSerializerLite` class which is used for login errors.  It exposes
some user information to the client, but nothing sensitive.

IssueID #82

### CODE REVIEW

**Comment by allynt** [comment]
We should hold off closing #82 until this code has been integrated into **orbis**.

**Comment by Reignable** [comment]
I'd say we can close #82 as this extra info won't break anything and there's no ticket for the orbis integration. That'll probably come when the frontend equivalent of `django-astrosat-users` is made.

File: astrosat_users/serializers/serializers_users.py [hunk 1]
```diff
@@ -13,8 +13,8 @@ class UserSerializerLite(serializers.ModelSerializer):
 
     class Meta:
         model = User
-        fields = ("email", "name", "username", "change_password")
-        read_only_fields = ("email", "name", "username", "change_password")
+        fields = ("email", "name", "username", "change_password", "is_verified", "is_approved", "accepted_terms")
+        read_only_fields = ("email", "name", "username", "change_password", "is_verified", "is_approved", "accepted_terms")
```
Comment (Reignable) [inline astrosat_users/serializers/serializers_users.py:17]: Do you need to have both `fields` and `read_only_fields`?

**Review by Reignable — APPROVED** [review]
Approved with one question but it's probably because I don't know Django

File: astrosat_users/serializers/serializers_users.py (hunk 1 above)
Comment (allynt) [reply astrosat_users/serializers/serializers_users.py:17]: `read_only_fields` protects me from ever using this serializer in a read-write context.

Comment (Reignable) [reply astrosat_users/serializers/serializers_users.py:17]: That makes sense but what I was asking was if a field is listed as a `read_only_field` does it _also_ need to be in `fields`?

Comment (allynt) [reply astrosat_users/serializers/serializers_users.py:17]: It's the other way around; every field that is part of the serializer needs to be in `fields`; those fields that are read-only should also be in `read_only_fields`.

**Pull request merged**